vinyl/tests/Functions.php

```diff
@@ -0,0 +1,42 @@
+<?php
+require_once '../functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class Functions extends TestCase {
+    public function testSuccessdisplayRecords()
+    {
+        //expected result of the test
+        $expected = '<div><h3>Cuthbert</h3><h4>Pizza</h4><p>7</p></div>';
```

Comment:
Don't forget to update your expected here now that you have tweaked the HTML generated by displayRecords()